index.html

```diff
@@ -9,36 +9,214 @@
-  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;600&display=swap" rel="stylesheet" >
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;600&display=swap" rel="stylesheet" crossorigin="anonymous" >
 </head>
 <body>
   <main>
-      <div class="header">
-       <a href="">My Logo</a>
-      <nav><a href=""><i class="fas fa-bars"></i></a></nav>
-      </div> 
+    <div class="header">
+      <a href="">My Logo</a>
+      <nav><a href="" aria-label="bars"><i class="fas fa-bars"></i></a></nav>
+    </div>
+
     <section class="container">
       <div class="intro">
         <h1>Hi! I'M DIRE</h1> 
           <h1>GLAD TO SEE <br>YOU!</h1>
-         <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <p>I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+             If you like what you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+      </div>
+
+      <div class="contact"> 
+        <a href="#" class="contacts">LET'S CONNECT</a>
       </div>
 
-    
-       <div class="contact"> 
-          <a href="#" class="contacts">LET'S CONNECT</a>
-        </div> 
       <div class="social-media">
         <ul class="sm">
-            <li><a href="#"><i class="fab fa-twitter"></i></a></li> 
-            <li><a href="#"><i class="fab fa-linkedin-in"></i></a></li>
-            <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
-            <li><a href="#"><i class="fab fa-github"></i></a></li> 
+          <li><a href="#" aria-label="twitter"><i class="fab fa-twitter"></i></a></li> 
+          <li><a href="#" aria-label="linkedin"><i class="fab fa-linkedin-in"></i></a></li>
+          <li><a href="#" aria-label="medium"><i class="fab fa-medium-m"></i></a></li>
+          <li><a href="#" aria-label="github"><i class="fab fa-github"></i></a></li> 
         </ul>
       </div>
     </section>
+      
+    <section>
+      <ul class="cards-container">
+        <li class="work one">
+        <img src="./snapshot-porfolio.png" alt="project-image" class="project-image1">
+        
+        <h3>TONIC</h3>
+
+        <div class="sub-title">
+          <h4>Canopy</h4>
+          <h4><i class="fas fa-circle"></i> Back End Dev</h4>
+          <h4><i class="fas fa-circle"></i> 2015</h4>
+        </div>
+
+        <section>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </section>
+
+        <ul class="languages">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+
+        <button class="see">See Project</button>
```

Comment:
- [ ] The button should follow the button styling: Active, Hover and Pressed (see design below) and kindly fix that.
![buttonsat](https://user-images.githubusercontent.com/65852621/133895354-5c6dcb0c-c735-42ba-8e61-d88ba93f479f.png)
 - [ ] The section should have the margins (top, bottom, left, right) defined in the design. Please checkout the difference below and kindly fix yours. 
 
 **Provided design**
![dire1](https://user-images.githubusercontent.com/65852621/133895517-3bb9b7bd-37e4-45b1-9393-8d0197c9fdea.png)

**Your implementation**
![dire2](https://user-images.githubusercontent.com/65852621/133895521-0f22a05e-b93f-4de8-9ecc-cb6a41e14ac3.png)